product-releases/2021-03-15-service-level-objectives.mdx

```diff
@@ -0,0 +1,45 @@
+---
+title: Service Level Objectives
+author: Daniel McSheehy
+author_title: Software Engineer
+author_url: https://github.com/DanielMcSheehy
+author_image_url: https://avatars.githubusercontent.com/u/28493438?s=460&u=4bbd1de69b9ca85d21e45350d15db419a27072fc&v=4
+tags: [service level objectives, standards]
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+Service Level Objectives (SLOs) allow you to define the metrics and values that matter for a service. 
+
+<!--truncate-->
+
+## All your SLOs for each service at a glance
+
+For each service you can view multiple SLO from integrations such as Datadog.
+
+Easily observing service level objectives keeps the high standard for availability within your organization.
+
+<img
+  alt='Service Level Objectives Popup'
+  src={useBaseUrl('img/service-level-objectives/popover.png')}
+/>
+
+## Include and set Standards based on Service Level Objectives
+
+With SLOs, we now enable you to create standards that allow you to easily identify which services 
+are not passing key measurements.
+
+Creating new standards based on services objectives, gives your organization insight on what actions need to be taken.
+
+<img
+  alt='SLO Standards'
+  src={useBaseUrl('img/service-level-objectives/standard-slo-criteria.png')}
+/>
+
+In addition, standards now allow you to observe which services are missing key service level objectives, which displays
+obvious indicators on where improvements can be made.
+
+<img
+  alt='SLO Standards'
+  src={useBaseUrl('img/service-level-objectives/service-slo-criteria.png')}
+/>
```

Comment:
below, can we remove the `.DS_Store` files? maybe we should add it to gitignore

Reply:
good call

Reply:
@DanielMcSheehy there's still 2 more ds store files i think